src/handlers/notify_zulip.rs

```diff
@@ -172,3 +173,69 @@ pub(super) async fn handle_input<'a>(
 
     Ok(())
 }
+
+fn get_team_to_be_nominated(labels: &Vec<Label>, msg: String) -> String {
+    let teams = labels
+        .iter()
+        .map(|label| &label.name)
+        .filter_map(|label| label.strip_prefix("T-"))
+        .collect::<Vec<&str>>();
+
+    // - If a single team label is found, replace the placeholder with that one
+    // - If multiple team labels are found and one of them is "compiler", pick that one
+    //   (currently the only team handling these Zulip notification)
+    // - else, return a generic text
+    if teams.len() == 1 {
+        msg.replace("{team}", teams[0])
+    } else if teams.contains(&"compiler") {
+        msg.replace("{team}", "compiler")
+    } else {
+        msg.replace(
+            "Needs `I-{team}-nominated`?",
```

Comment:
This is hardcoding a non-replacement string -- should we really do that? I would expect this trailing case to be just `msg` I think, i.e., no replacement is performed.

Reply:
> This is hardcoding a non-replacement string -- should we really do that? I would expect this trailing case to be just `msg` I think, i.e., no replacement is performed.

I thought a reasonable default message would be better than ``Needs `I-{team}-nominated`?`` which feels like someone forgot to localize a string :-)

But I am also ok with leaving it as it is. I'm sure nobody reads it anyway :smile: